recipe/opencv_persistence_data_member.patch

```diff
@@ -0,0 +1,199 @@
+--- modules/core/src/persistence.cpp
++++ modules/core/src/persistence.cpp
+@@ -1367,7 +1367,7 @@
+             CV_PARSE_ERROR("Unrecognized Base64 header");
+ 
+         std::vector<char> header(base64::HEADER_SIZE + 1, ' ');
+-        base64::base64_decode(beg, header.data(), 0U, base64::ENCODED_HEADER_SIZE);
++        base64::base64_decode(beg, &header.front(), 0U, base64::ENCODED_HEADER_SIZE);
```

Comment:
Actually it looks like our own patches are to blame. Could we please try dropping this patch, @mariusvniekerk? If we do need it, we will need to come up with something else.